addons/mail/static/src/components/message/message.xml

```diff
@@ -66,7 +66,7 @@
                         <t t-if="!messageView.isSquashed">
                             <div class="o_Message_header d-flex flex-wrap align-items-baseline ml-2">
                                 <t t-if="messageView.message.author">
-                                    <strong class="o_Message_authorName o_Message_authorRedirect o_redirect o_cursor_pointer text-truncate" role="link" tabindex="0" t-on-click="messageView.onClickAuthorName" title="Open profile">
+                                    <strong class="o_Message_authorName o_Message_authorRedirect o_redirect o_cursor_pointer text-truncate" role="link" tabindex="0" t-on-click="messageView.onClickAuthorName" title="Open Chat">
```

Comment:
Similarly to what is done for the avatar, the link and all related attributes should be deactivated when `hasAuthorOpenChat` is false.